### PR TITLE
some fixes for pycodestyle E221

### PR DESCRIPTION
**Pull request by fchapoton**
this is fixing some of the warnings

`E221 multiple spaces before operator`

mostly scripted with autopep8, plus a few manual cases

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about.

### CODE REVIEW

File: src/sage/crypto/sboxes.py [hunk 1]
```diff
@@ -1433,7 +1433,7 @@ def monomial_function(n, e):
     20,26,7,31,19,12,10,15,22,30,13,14,4,24,9,
     18,27,11,1,21,6,16,2,28,23,5,8,3,0,17,29,25])
 
-Shamash  = SBox([16, 14, 13, 2, 11, 17, 21, 30, 7, 24, 18, 28, 26, 1, 12, 6,
+Shamash = SBox([16, 14, 13, 2, 11, 17, 21, 30, 7, 24, 18, 28, 26, 1, 12, 6,
                  31, 25, 0, 23, 20, 22, 8, 27, 4, 3, 19, 5, 9, 10, 29, 15])
```
Comment (mkoeppe) [inline src/sage/crypto/sboxes.py]: repair indentation please

File: src/sage/interfaces/tides.py [hunk 2]
```diff
@@ -831,13 +831,13 @@ def genfiles_mpfr(integrator, driver, f, ics, initial, final, delta,
         elif el[0] == 'div_c':
             string += 'div_t_cv(itd, "' + el[2] + '", ' + el[1] + ', link[{}], i);'.format(i)
         elif el[0] == 'log':
-            string += 'log_t(itd, ' + el[1]  + ', link[{}], i);'.format(i)
```
Comment (mkoeppe) [inline src/sage/interfaces/tides.py:834]: switch to f-strings while at it?

**Review by soehms — CHANGES_REQUESTED** [review]
The following doctest failure seems to be related to your previous commit:

```
File "src/sage/interfaces/tides.py", line 695, in sage.interfaces.tides.?
Failed example:
    l[30]
Expected:
    '\t\tmpfrts_pow_t_c(itd, link[2], "-1.500000000000000000000000000000000000000000000000000", link[3], i);\n'
Got:
    '\t\tmpfrts_pow_t_c(itd, link[2], "-1.500000000000000000000000000000000000000000000000000", link[{i}], i);\n'
**********************************************************************
File "src/sage/interfaces/tides.py", line 736, in sage.interfaces.tides.?
Failed example:
    l[30]
Expected:
    '\t\tmpfrts_pow_t_c(itd, link[2], "-1.500000000000000000000000000000000000000000000000000", link[3], i);\n'
Got:
    '\t\tmpfrts_pow_t_c(itd, link[2], "-1.500000000000000000000000000000000000000000000000000", link[{i}], i);\n'
**********************************************************************
1 item had failures:
   2 of  95 in sage.interfaces.tides.?
    [120 tests, 2 failures, 0.28 s]
```

**Review by soehms — CHANGES_REQUESTED** [review]
Now there is this one:

```
File "src/sage/interfaces/tides.py", line 691, in sage.interfaces.tides.?
Failed example:
    l[15]
Expected:
    '\tstatic int PARAMETERS = 0;\n'
Got:
    '\tstatic int PARAMETERS = 4;\n'
**********************************************************************
1 item had failures:
   1 of  95 in sage.interfaces.tides.?
    [120 tests, 1 failure, 0.09 s]
```

Probably caused by this change (`PAR` -> `VAR`):

```
-    outfile.write("\tstatic int PARAMETERS = {};\n".format(PAR))
...
+        outfile.write(f"\tstatic int PARAMETERS = {VAR};\n")
```

@miguelmarco, ~can you check that the generated files are still good for the *TIDES* library?~ (not necessary any more). 

BTW: The workflows that run doctest of optional and experimental packages seem to be broken (at least the ones that run for the tag 10.2.beta9, example for [Ubuntu jammy maximal](https://github.com/sagemath/sage/actions/runs/6700323366/job/18294092977) is failing [here](https://github.com/sagemath/sage/actions/runs/6700323366/job/18294092977#step:10:37)).

Is this a known issue?

**Review by soehms — APPROVED** [review]
LGTM!

The generated files of the examples are not changed by the branch.

Anyway, the idea behind E221 is not clear to me. Looking through the changes my impression is that it makes code less readable.

**Pull request merged**